OSDOCS-16959-ldap-identity-sync CQA (#116487)
openshift/openshift-docs: modules/ldap-syncing-pruning.adoc

//
// * authentication/ldap-syncing.adoc

:_mod-docs-content-type: PROCEDURE
[id="ldap-syncing-pruning_{context}"]
= Running a group pruning job

[role="_abstract"]
Run a group pruning job to remove LDAP-synced groups from {product-title} when they no longer exist on your LDAP server so you can keep cluster group records aligned with your directory.

> An administrator can also choose to remove groups from {product-title} records if the records on the LDAP server that created them are no longer present. The prune job accepts the same sync configuration file and allowlists or denylists as used for the sync job.

Reviewer: Do we need _An administrator can also choose to remove groups from {product-title} records if the records on the LDAP server that created them are no longer present._ here? Is "run a group pruning job" and "remove groups from {product-title} records" the same task? 
```suggestion
Run a group pruning job to remove LDAP-synced groups from {product-title} when they no longer exist on your LDAP server so you can keep cluster group records aligned with your directory.

The prune job accepts the same sync configuration file and allowlists or denylists as used for the sync job.
```

.Procedure

> . To prune groups using a sync configuration file, run the following command:

Reviewer: Does the user perform these steps in order? If not, use bullets for unordered list.

```suggestion
* To prune groups using a sync configuration file, run the following command:
```

Author: I also removed the "To" and started with "Prune"

+
[source,terminal]
----
$ oc adm prune groups --sync-config=/path/to/ldap-sync-config.yaml --confirm
----

> . To prune groups using an allowlist file, run the following command:

Reviewer: ??

```suggestion
* To prune groups using an allowlist file, run the following command:
```

+
[source,terminal]
----
$ oc adm prune groups --whitelist=/path/to/whitelist.txt --sync-config=/path/to/ldap-sync-config.yaml --confirm
----

> . To prune groups using a denylist file, run the following command:

Reviewer: ??

```suggestion
* To prune groups using a denylist file, run the following command:
```
